schemas/package.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: name
+      level: extended
+      type: keyword
+      description: Package name
+      example: go
+
+    - name: version
+      level: extended
+      type: keyword
+      description: Package version
+      example: 1.12.9
+
+    - name: description
+      level: extended
+      type: keyword
+      description: Description of the package.
+      example: Open source programming language to build simple/reliable/efficient software.
+
+    - name: size
+      format: string
+      level: extended
+      type: long
+      description: Package size in bytes.
+      example: 62231
+
+    - name: installed
+      level: extended
+      type: date
+      description: Time when package was installed.
+
+    - name: type
+      level: extended
+      type: keyword
+      description: Information about package type. When installed from a package manager, this would contain the
+        package manager name.
+      example: npm
```

Comment:
In the spirit of keeping the examples consistent from one another, please put an example here that works with the "golang" example used throughout the field set. So perhaps put example "rpm" or "homebrew".